beacon-chain/rpc/beacon_server.go

```diff
@@ -216,39 +282,17 @@ func (bs *BeaconServer) BlockTreeBySlots(ctx context.Context, req *pb.TreeBlockS
 	}, nil
 }
 
-// TODO(#2307): Refactor for v0.6.
-// nolint
-func (bs *BeaconServer) defaultDataResponse(ctx context.Context, currentHeight *big.Int, eth1FollowDistance int64) (*pbp2p.Eth1Data, error) {
+func (bs *BeaconServer) defaultEth1DataResponse(ctx context.Context, currentHeight *big.Int, eth1FollowDistance int64) (*pbp2p.Eth1Data, error) {
```

Comment:
Some commentary on what the default response would be a good idea.

Reply:
added